packages/dev-server/index.js

```diff
@@ -33,7 +32,7 @@ module.exports = (neutrino, opts = {}) => {
       historyApiFallback: true,
       publicPath: '/',
       headers: {
-        host: `${publicHost}:${port}`
+        host: publicHost
```

Comment:
What does this produce when no value is passed for `public`? What is produced here when the host is overridden?

Reply:
@eliperelman When no value is passed to `public` and the `host` is overridden:

```
module.exports = {
  use: [
    [
      '@neutrinojs/react',
      {
        devServer: {
          host: '0.0.0.0',
          port: 3000,
        }
      }
    ]
  ]
};
```

Outputs in the inspect mode (relevant snippet):

```
{
  devServer: {
    headers: {
      host: '0.0.0.0'
    },
    host: '0.0.0.0',
    port: 3000,
    'public': '0.0.0.0',
  },
  entry: {
    index: [
      '/Users/haali/Documents/Mozilla/projects/neutrino-dev/node_modules/webpack-dev-server/client/index.js?http://0.0.0.0',
      '/Users/haali/Documents/Mozilla/projects/neutrino-dev/node_modules/webpack/hot/dev-server.js',
      '/Users/haali/Documents/Mozilla/projects/neutrino-dev/node_modules/react-hot-loader/patch.js',
      '/Users/haali/Documents/Mozilla/projects/todelete/src/index'
    ]
  },
```

Reply:
What if public is `0.0.0.0:3000`?